py/src/proving_systems/stark/mod.rs

```diff
@@ -0,0 +1,79 @@
+use lambdaworks_stark::FE;
+
+use pyo3::class::basic::CompareOp;
+use pyo3::exceptions::PyValueError;
+use pyo3::types::*;
+use pyo3::*;
+
+use crate::math::unsigned_integer::element::PyU256;
+
+#[pyclass(name = "FieldElement")]
+#[derive(Clone)]
+pub struct PyFieldElement(pub FE);
+
+#[pymethods]
+impl PyFieldElement {
+    #[new]
+    pub fn new(value: &PyU256) -> Self {
+        Self(FE::new(value.0))
+    }
+
+    fn __richcmp__(&self, other: PyRef<Self>, op: CompareOp) -> Py<PyAny> {
+        let py = other.py();
+        match op {
+            CompareOp::Eq => (self.0 == other.0).into_py(py),
+            CompareOp::Ne => (self.0 != other.0).into_py(py),
+            _ => py.NotImplemented(),
+        }
+    }
+
+    fn __add__(&self, other: &Self) -> Self {
+        Self(&self.0 + &other.0)
+    }
+
+    fn __sub__(&self, other: &Self) -> Self {
+        Self(&self.0 - &other.0)
+    }
+
+    fn __mul__(&self, other: &Self) -> Self {
+        Self(&self.0 * &other.0)
+    }
+
+    fn __truediv__(&self, other: &Self) -> PyResult<Self> {
+        match other.0 == FE::zero() {
```

Comment:
I'm assuming the following method exists:
```suggestion
        match other.0.is_zero() {
```

Reply:
I didn't found that method. Maybe we can implement it in the future.